install/kubernetes/helm/istio/values.yaml

```diff
@@ -155,6 +155,12 @@ global:
   # and this options must be set off.
   crds: true
 
+  # Namespace in which Mixer service is running. Mandatory if Gateways are running in another namespace.
```

Comment:
Please don't, too many options already. Control plane will be all in one namespace.